.NET/Microsoft.Recognizer.Text.TimeZone/Extractors/IExtractor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Recognizers.Text.TimeZone
+{
+    public interface IExtractor
```

Comment:
We already have an IExtractor interface in the system, please reuse than to the extent possible.